content/v1.3/introduction/getting-started.md

```diff
@@ -549,8 +554,10 @@ RSpec.describe 'List books' do
 end
 ```
 
-We create the required records in our test and then assert the correct number of book classes on the page.
-When we run this test, it should pass. If it does not pass, a likely reason is that the test database was not migrated.
+We create a single Book record in our test and then assert that the title and author are displayed on the page.
+When we run this test, it will fail.
+This is because we haven't updated our template yet,
+it still has the hard-coded books from earlier.
```

Comment:
Updated this to "We create a single Book record in our test and then expect that the title and author are displayed on the page. This test fails since we haven't updated our template yet, and it still includes the hard-coded books from earlier."

What do you think of that?